layers/+themes/dark-window/packages.el

```diff
@@ -0,0 +1,28 @@
+;;; packages.el --- dark-window layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Oleg Pykhalov <go.wigust@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst dark-window-packages
+  '(frame-cmds))
+
+(defun dark-window/init-frame-cmds ()
+  (use-package frame-cmds
+    :config
+    (progn
```

Comment:
This whole `progn` is a direct copy of http://nicolas.petton.fr/blog/emacs-dark-window-decoration.html, so you should a comment attributing credit to the original author of the code.